api/resources/serializers.py

```diff
@@ -60,3 +62,15 @@ def get_absolute_url(self, obj):
                 'resource_id': obj._id,
             },
         )
+
+    def create(self, validated_data):
+        # Already loaded by view, so no need to error check
+        guid = self.context['request'].data['registration']
+        primary_registration = Registration.load(guid)
+
+        try:
+            root_outcome = Outcome.objects.for_registration(primary_registration, create=True)
+        except NoPIDError:
+            raise Conflict('Cannot add Resrouces to a Registration without a DOI')
```

Comment:
```suggestion
            raise Conflict('Cannot add Resources to a Registration that does not have a DOI')
```

Two things here: Resources was misspelled, and I'm guessing the registration is the thing that needs the DOI, so this clarifies that a bit.